### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by Adarsh321123** [lean-dojo/LeanDojo#210] [General]
I have some general questions:
1. What's the roadmap for LeanDojo?
2. How often do you fix bugs?
3. How often do you support new Lean versions?

**Comment by yangky11** [comment]
LeanDojo is a community-driven open-source project, and as such, there isn't an official roadmap. My collaborators and I have projects that may be related to LeanDojo to some extent, but these projects are ongoing, and we will only be able to share details when they become more mature.

Given that Lean is constantly evolving and frequently makes breaking changes, it can be challenging to keep LeanDojo up to date. Bug fixes and support for new Lean versions are also driven by the community. Due to my limited bandwidth, I plan to prioritize fixing issues that are roadblocks for many users and facilitating contributions from the community. Unfortunately, LeanDojo does not have a dedicated on-call support team.
